pyproject.toml

```diff
@@ -16,7 +17,7 @@ classifiers = [
 authors = [
   { name="Scientific Software Center", email="ssc@uni-heidelberg.de" },
 ]
-version = "0.0.1"
+dynamic = ["version"]
 
 dependencies = [
```

Comment:
`jupyter` is also a dependency for the users, if they are going to run this through a notebook - maybe could be added to the `[viz]` block?